Shared/Extensions/URLExtensions.swift

```diff
@@ -389,6 +389,14 @@ extension URL {
         }
         return nil
     }
+    
+    public var isMediaSiteURL: Bool {
```

Comment:
Should probably say this is rewards specific somehow. Since there are obviously tons of other media URLs.